Packs/CommonScripts/Scripts/ZipStringsArrays/ZipStringsArrays.yml

```diff
@@ -0,0 +1,35 @@
+commonfields:
+  id: ZipStrings
+  version: -1
+name: ZipStrings
+script: ''
+type: python
+tags: []
```

Comment:
Wouldn't it make sense to make this a transformer?

Reply:
it will, but not efficient. here is an example to make it clear:
list1 = ['a','b']
list2 = ['c','d']
when running as transformer on list1 with list2 as param, the script get the following arg at first run:
{'value':'a', list2:['c','d']}
the second run: 
{'value':'b':list2:['c','d']}

meaning in order to create the correct combination (without dups or missing results) we need to check for each run which index are we on and take the matching from list2.
While this is possible (by getting all the params at once, and taking for each run the index to get the correct value from list2) it is both overkill, and very specific (we need only zip when using script, and a complex logic for a transformer). I believe a script is a much better solution.